docs/getting-started/linux-installer.md

```diff
@@ -8,6 +8,11 @@ script deploys and configures:
 - Splunk OpenTelemetry Collector for Linux
 - [SignalFx Smart Agent and collectd bundle](https://github.com/signalfx/signalfx-agent/releases) (**x86_64/amd64 platforms only**)
 - [Fluentd (via the TD Agent)](https://www.fluentd.org/)
+  - Optional, **enabled** by default
+  - See the [Fluentd Configuration](#fluentd-configuration) section for more details or how to disable installation
+- [Splunk OpenTelemetry Auto Instrumentation for Java](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#linux-java-auto-instrumentation)
+  - Optional, **disabled** by default
+  - See the [Auto Instrumentation](#auto-instrumentation) section for how to enable installation and more details
```

Comment:
Suggestion: "See the auto-instrumentation section for the installation process."